src/services/PhotoService.ts

```diff
@@ -17,4 +18,23 @@ export default class PhotoService implements IService<Photo>{
     console.log(payload);
     return PhotoService.getRepository().save({ ...payload });
   }
+
+  public async updateById(id:number, payload: PhotoUpdateValidator): Promise<Photo> {
+    const data = await PhotoService.getRepository().findById(id);
+    if(!data){
```

Comment:
In those cases, it's better if we throw a descriptive exception, otherwise, any other component that wants to use this functionality will need to figure out a way to handle `null` values on its own, which is far from being descriptive and makes the program harder to understand!

By throwing exceptions, we bubble up the error and we only need a middleware or wrapper that's able to catch exceptions, therefore, we don't need other components using this one to handle the error on their own.

Please check my current master branch and how error handling middleware works and how we can throw exceptions from services.